components/camel-olingo2/camel-olingo2-api/pom.xml

```diff
@@ -122,7 +122,7 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-javadoc-plugin</artifactId>
                         <configuration>
-                            <additionalparam>-Xdoclint:none</additionalparam>
+                            <doclint>none</doclint>
```

Comment:
This seems wrong, please rebuild the whole project

Reply:
Agreed. Thanks. 
I will remove these changes.